server/services/sequence_service.go

```diff
@@ -97,7 +97,7 @@ func (s *sequenceServerImpl) AttemptSequence(ctx context.Context, in *pb.Attempt
 	if err != nil {
 		return nil, status.Errorf(
 			codes.Internal,
-			err.Error(),
+			"%s", err.Error(),
```

Comment:
`err.Error()` returns a string, so we don't need this format string. Below as well.